libafl/src/corpus/cached.rs

```diff
@@ -125,14 +119,14 @@ where
     #[inline]
     fn get(&self, idx: CorpusId) -> Result<&RefCell<Testcase<I>>, Error> {
         let testcase = { self.inner.get(idx)? };
-        self.cache_testcase(testcase, idx, false)?;
+        self.cache_testcase(testcase, idx)?;
         Ok(testcase)
     }
     /// Get by id; considers both enabled and disabled testcases
     #[inline]
     fn get_from_all(&self, idx: CorpusId) -> Result<&RefCell<Testcase<Self::Input>>, Error> {
```

Comment:
Should we even cache if the testcase is disabled? I don't know if we usually want that, it should be used way less?